### PR TITLE
Hamster Stake Pool added

### PR DESCRIPTION
**Pull request by MagisterTechnicus**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -3372,5 +3372,19 @@
...
+    "ticker": "HAMDA",
+    "poolId": "bb32c21c1ca7dc8f73fad1b6263987139a868fad820b3c3069fefcd6",
+    "social": {
+      "website": "https://hamster-stake-pool.de",
+      "twitter": "HAMDAStakePool",
+      "facebook": "",  
+      "instagram": "",
+      "youtube": "https://www.youtube.com/channel/UCVchFtu9uBRnnkxAye31etw"
+    },
+    "telegram": "HamsterStakePool",
```
Comment (leo42) [inline registry.json:3386]: Please use a personal telegram handle , no groups or pages.

**Review by leo42 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (Bez625) [reply registry.json]: You have removed the telegram handle. You need to supply a TG handle, but it cannot be a group or page.

**Comment by MagisterTechnicus** [comment]
Sorry, didn't know that. Done.

**Comment by MagisterTechnicus** [comment]
Hope everything is correct now

**Pull request merged**